0.10/Dockerfile

```diff
@@ -30,6 +30,10 @@ COPY ./s2i/bin/ $STI_SCRIPTS_PATH
 # run and build the applications.
 COPY ./contrib/ /opt/app-root
 
+# Add ./node_modules/.bin to the $PATH, so users won't need to use
+# --global installation (npm i -g) to make npm scripts available on the CLI
+RUN echo "export PATH=\"\$PATH:\$HOME/node_modules/.bin\"" >> /etc/profile.d/custom.sh
```

Comment:
Thanks for the feedback - figured there might be a better way.  I've fixed and rebased